indico/modules/events/registration/models/items.py

```diff
@@ -285,6 +285,10 @@ class RegistrationFormItem(db.Model):
     # - registration_form (RegistrationForm.form_items)
 
     @property
+    def filter_choices(self):
```

Comment:
I meant the `RegistrationFormFieldBase` base class when i suggested to move it to the base field class. it doesn't really belong here and besides that you never access it through any of the item classes anyway.